robotbase/src/main/java/com/team2357/frc2022/Robot.java

```diff
@@ -30,6 +31,7 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+    LiveWindow.disableAllTelemetry();
```

Comment:
Was this what was causing the loop overruns, and the driver station to lose connection?

Reply:
I'm not sure, it's a theory